kernel/classes/ezcontentobjecttreenodeoperations.php

```diff
@@ -113,6 +113,7 @@ static function move( $nodeID, $newParentNodeID )
                 $nodeIDList = array( $nodeID );
                 eZSearch::removeNodeAssignment( $node->attribute( 'main_node_id' ), $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList );
                 eZSearch::addNodeAssignment( $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList, true );
+                eZSearch::refreshChildren( $node );
```

Comment:
ah, right, my bad, I didn't check what is actually happening in cronjob task :(
Once all is configured well, meaning `DelayedIndexing` is enabled and cronjob is setup there should be no issue and this is not needed anymore. 

But from the other hand, assuming that `DelayedIndexing` is disabled (it is disabled by default), without adding this `refreshChildren` call there will be no reindex of children at all and this looks like an issue (it was reported after all, that's where this PR comes from). 

There are probably few ways to fix it:
1. "Force" users to use delayed indexing / inform them that there will be no reindex if they don't
2. Keep `refreshChildren` method but in this case we need to do something with the way it is called as currently it adds task to reindex every single node of subtree and cronjob takes each of such node and reindexes whole subtree so there's a lot of redundancy here as single node can be reindexed multiple times